examples/service_worker/src/lib.rs

```diff
@@ -273,12 +274,8 @@ fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
                 .expect("get `PushManager`");
             // @TODO: Replace the call to `subscribe` with the web_sys equivalent when wasm_bindgen releases a version >= 0.2.68.
             orders.perform_cmd(async move {
-                Msg::Subscribed(
-                    subscribe(&push_manager, key)
-                        .await
-                        .into_serde()
-                        .map_err(ServiceWorkerError::SerdeJson),
-                )
+                let js = subscribe(&push_manager, key).await;
```

Comment:
Optional: See the `TODO` two lines above. Then we can remove the function `subscribe`.

Reply:
I will leave this for follow-up work.